setup.py

```diff
@@ -12,7 +12,7 @@
     include_dirs  = ['http-parser', '/usr/include/libev'],
     define_macros = [('WANT_SENDFILE', '1'),
                      ('WANT_SIGINT_HANDLING', '1')],
-    extra_compile_args = ['-std=c99', '-fno-strict-aliasing', '-fcommon',
+    extra_compile_args = ['-std=c11', '-fno-strict-aliasing', '-fcommon',
```

Comment:
The Makefile doesn't work at all from my side. And if in setup.py I do not enable c11 I got a warning:
x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -g -fdebug-prefix-map=/build/python3.5-7CCmgg/python3.5-3.5.3=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DWANT_SENDFILE=1 -DWANT_SIGINT_HANDLING=1 -Ihttp-parser -I/usr/include/libev -I/usr/include/python3.5m -c bjoern/portable_sendfile.c -o build/temp.linux-x86_64-3.5/bjoern/portable_sendfile.o -std=c99 -fno-strict-aliasing -fcommon -fPIC -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -g
In file included from /usr/include/python3.5m/Python.h:65:0,
                 from bjoern/portable_sendfile.h:1,
                 from bjoern/portable_sendfile.c:3:
/usr/include/python3.5m/pytime.h:136:56: warning: ‘struct timespec’ declared inside parameter list will not be visible outside of this definition or declaration
 PyAPI_FUNC(int) _PyTime_AsTimespec(_PyTime_t t, struct timespec *ts);
                                                        ^~~~~~~~